libraries/botbuilder-lg/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "botbuilder-lg",
+  "description": "Typescript version of Microsoft.Expression from C# version (https://github.com/Microsoft/botbuilder-dotnet/tree/ComposableDialog/libraries/Microsoft.CommonExpressions) ",
+  "version": "4.1.6",
+  "license": "MIT",
+  "keywords": [
+    "botbuilder",
+    "botframework",
+    "expression"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/botbuilder-js.git"
+  },
+  "main": "./lib/index.js",
+  "typings": "./lib/index.d.ts",
+  "dependencies": {
+    "antlr4ts": "0.5.0-alpha.1",
+    "botframework-expressions": "4.1.6",
+    "lodash": "^4.17.11",
+    "uuid": "^3.3.3"
```

Comment:
since the dialog system will have a dependency on these files we should ensure that they can be loaded into a browser.